conans/client/build/cmake_flags.py

```diff
@@ -168,6 +168,8 @@ def _cmake_cross_build_defines(self):
                         ret["CMAKE_SYSTEM_NAME"] = "Generic"
         if os_ver:
             ret["CMAKE_SYSTEM_VERSION"] = os_ver
+            if str(os_).lower() == "macos":
```

Comment:
why the lower? the setting is standardized as "Macos"

Reply:
It was a copy/paste from https://github.com/conan-io/conan/blob/4a8a8d88c5814dcc985c699bfef298c8a5e2f48c/conans/client/build/cmake_flags.py#L240, let's use it like `Macos` and see what happens